.github/workflows/wheels.yaml

```
@@ -16,7 +16,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ "ubuntu-20.04", "macos-latest" ]
```

Would building on Ubuntu 22.04 already work? That would give the advantage of newer compilers.
```suggestion
        os: [ "ubuntu-22.04", "macos-latest" ]
```